lib/peanut.dart

```diff
@@ -32,7 +32,26 @@ Future<Null> run(String targetDir, String targetBranch, String commitMessage,
       await Directory.systemTemp.createTemp('peanut.$secondsSinceEpoch.');
 
   try {
-    var args = ['build', '--output', tempDir.path, targetDir, '--mode', mode];
+    var args = useBuildRunner
+        // Build with build_runner.
+        // TODO Clean this up when better command line args exist.
+        ? [
+            'run',
+            'build_runner',
+            'build',
+            '--output',
+            tempDir.path,
+            // Force build with dart2js instead of dartdevc.
+            '--define',
+            '"build_web_compilers|entrypoint=compiler=dart2js"',
+            // Use checked mode and minify.
+            // TODO Add --no-source-maps flag when fix is published.
+            '--define',
+            '"build_web_compilers|entrypoint=dart2js_args=[\"--checked\", '
```

Comment:
Not checked!

Reply:
OK, any flags that you want? --trust-type-annotations?